scripts/seal.coffee

```diff
@@ -0,0 +1,9 @@
+module.exports = (robot) ->
+  robot.hear /seal me up/i, (res) ->
```

Comment:
I would add `^` to the regular expression `/^seal me up/i`so it doesn't catch it in a middle of a phrase.

Although I'm not sure where someone would use that in a middle of a sentence... 

`Last night I seal me up` :D